actions/copy/worker.js

```diff
@@ -130,7 +150,9 @@ async function floodgateContent(spToken, adminPageUri, projectExcelPath, project
     logger.info('Project excel file updated with copy status.');
 
     if (failedCopies.length > 0 || failedPreviews.length > 0) {
-        logger.info('Error occurred when floodgating content. Check project excel sheet for additional information.');
+        const errorMessage = 'Error occurred when floodgating content. Check project excel sheet for additional information.';
+        logger.info(errorMessage);
+        throw new Error(errorMessage);
```

Comment:
Perhaps this is where `COMPLETED_WITH_ERROR` can be set. Right now, even if error occurred here, the status is set to `COMPLETED`.